### PR TITLE
add TW (taiwan) source + support for headers

### PR DESCRIPTION
**Pull request by camjc**
## Summary

Add TW source.
@ryanblock  adds support for headers here.

### CODE REVIEW

**Comment by camjc** [comment]
dash3 has cumulative data but needs header
```
curl 'https://covid19dashboard.cdc.gov.tw/dash3' -H 'Origin: https://919644827-atari-embeds.googleusercontent.com'
```

File: src/shared/sources/tw/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  country,
+  priority: 1,
+  friendly: {
+    name: 'Taiwan CDC',
+    url: 'https://cdc.gov.tw/'
+  },
+  maintainers: [ maintainers.camjc ],
+  scrapers: [
+    {
+      startDate: '2020-01-15',
```
Comment (camjc) [inline src/shared/sources/tw/index.js:25]: Actually I dunno when this goes back to. Should we set today/nothing/pick something?

Comment (ryanblock) [reply src/shared/sources/tw/index.js:25]: Today: if it is timeseries we will never get data before the date you set. If not timeseries, date of merge is fine.

Comment (camjc) [inline src/shared/sources/tw/index.js:25]:
```suggestion
      startDate: '2020-05-05',
```

**Pull request merged**